### PR TITLE
BUGFIX: SD-90 remove top number on yaxis of heatmap

### PR DESCRIPTION
**Pull request by lanhq**

### CODE REVIEW

**Comment by lanhq** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/362/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1138/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/174/ : UNSTABLE in 15m 12s

**Comment by lanhq** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/363/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1139/ : SUCCESS in 20s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/175/ : SUCCESS in 9m 47s

File: src/components/visualizations/chart/highcharts/heatmapConfiguration.ts [hunk 1]
```diff
@@ -41,6 +41,27 @@ const HEATMAP_TEMPLATE = {
...
         borderWidth: 0,
         nullColor: 'url(#empty-data-pattern)'
+    }],
+    yAxis: [{
+        labels: {
+            formatter() {
+                const  axis = this.axis;
+                const tickPositions = axis.tickPositions;
+                const isLast = this.isLast;
+                const lastIndex = tickPositions[tickPositions.length - 1];
```
Comment (ivan-nejezchleb) [inline src/components/visualizations/chart/highcharts/heatmapConfiguration.ts]: use lodash 'last' function for that

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1142/ : SUCCESS in 33s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/659/ : FAILURE in 2m 04s

File: src/components/visualizations/chart/highcharts/heatmapConfiguration.ts [hunk 2]
```diff
@@ -41,6 +42,27 @@ const HEATMAP_TEMPLATE = {
     series: [{
         borderWidth: 0,
         nullColor: 'url(#empty-data-pattern)'
+    }],
+    yAxis: [{
+        labels: {
+            formatter() {
+                const  axis = this.axis;
```
Comment (ivan-nejezchleb) [inline src/components/visualizations/chart/highcharts/heatmapConfiguration.ts]: use const { axis, isLast } = this

File: src/components/visualizations/chart/highcharts/heatmapConfiguration.ts [hunk 3]
```diff
@@ -41,6 +42,27 @@ const HEATMAP_TEMPLATE = {
...
+                const tickPositions = axis.tickPositions;
```
Comment (ivan-nejezchleb) [inline src/components/visualizations/chart/highcharts/heatmapConfiguration.ts]: use const { tickPositions, categories } = axis

File: src/components/visualizations/chart/highcharts/heatmapConfiguration.ts [hunk 4]
```diff
@@ -41,6 +42,27 @@ const HEATMAP_TEMPLATE = {
     series: [{
         borderWidth: 0,
         nullColor: 'url(#empty-data-pattern)'
+    }],
+    yAxis: [{
```
Comment (ivan-nejezchleb) [inline src/components/visualizations/chart/highcharts/heatmapConfiguration.ts]: try to add storybook for this case

**Comment by lanhq** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1161/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/675/ : FAILURE in 2m 13s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/375/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1164/ : SUCCESS in 47s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/184/ : UNSTABLE in 9m 57s

**Comment by lanhq** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1168/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/679/ : SUCCESS in 2m 11s

**Comment by gdgate** [comment]
Build in pipeline check-extended aborted.

**Comment by lanhq** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1170/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/680/ : SUCCESS in 1m 32s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/379/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1171/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/188/ : UNSTABLE in 11m 57s

**Comment by lanhq** [comment]
extended test - storybook

File: src/components/visualizations/chart/highcharts/heatmapConfiguration.ts (hunk 4 above)
Comment (lanhq) [reply src/components/visualizations/chart/highcharts/heatmapConfiguration.ts]: done, storybook added

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/385/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1180/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/193/ : SUCCESS in 10m 05s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/237/ : SUCCESS in 13s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/689/ : SUCCESS in 2m 15s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/387/ : SUCCESS in 8m 20s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/134/ : SUCCESS in 10m 59s

**Pull request merged**